### DISCUSSION TITLE
Installation stops at 6/9 during compiling

### DISCUSSION THREAD

**Discussion by adbasanta** [chatwoot/chatwoot#6117] [Self hosted] [answered]
Hi to all,

Does anyone have experienced the same issue I am facing? I'm stuck in installation in steps 6/9 during compiling stage as below.
Does anyone know this issue? It seems related to browserslist but when I try to update it nothing happens.
Thanks for helping.
![log](https://user-images.githubusercontent.com/89345000/208864247-371de8cd-8e26-4105-9cf2-a93a2e9b8829.PNG)

**Comment by Aghiles97** [comment, marked as answer]
Hello, I had this issue no matter how many times I have created a new Ubuntu instance and installed again, to fix this, I had to increase the server ram from 2 to 4 go and CPU count from 1 to 2, and it worked fine this time.

**Comment by adbasanta** [comment]
thanks a lot. it worked.

**Reply by Aghiles97** [reply]
Glad I could help.
